### ISSUE TITLE
Incorrect usage Get-Random in the templates [person*]

### ISSUE THREAD

**Issue by it-praktyk** [dfinke/NameIT#2]
I've read the blog post about Get-Random strange behaviour/design decisssion http://tfl09.blogspot.com/2016/01/powershells-get-random-cmdleta-curiosity.html.

```powershell
Get-Help Get-Random -full

<part>
 -Maximum [<Object>]
     Specifies a maximum value for the random number. Get-Random returns a value that is less than the maximum (not equal).
</part>
```

The current implementation never doesn't return the last positions from the last/first names lists.

I'll prepare the updated soon.

**Comment by dfinke** [comment]
Yes, didn't catch that. Looking forward to the update.

**Comment by it-praktyk** [comment]
Corrected in the pull request #3
